bottlecap/src/proc/mod.rs

```diff
@@ -0,0 +1,82 @@
+pub mod constants;
+
+use std::{
+    error::Error,
+    fs::File,
+    io::{self, BufRead},
+};
+
+use constants::{LAMDBA_NETWORK_INTERFACE, PROC_NET_DEV_PATH};
+
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub struct NetworkData {
+    pub rx_bytes: f64,
+    pub tx_bytes: f64,
+}
+
+pub fn get_network_data() -> Result<NetworkData, Box<dyn std::error::Error>> {
+    get_network_data_helper(PROC_NET_DEV_PATH)
+}
```

Comment:
Is this method really needed?

Reply:
This was following the pattern from the agent because we use the inner method for unit testing

Reply:
Rename the helper then to `get_network_data_from_path`

Reply:
Actually, why would we even need to have the `get_network_data`, can't we just use the `get_network_data_from_path`?

Reply:
I guess it isn't necessary, but we use the get_network_data function in two different files, so I'm not sure if we should be importing and passing in that constant in both

Reply:
Fair point, I'll leave the decision to you